website_docs/manual.md

```diff
@@ -46,10 +46,13 @@ func newExporter(ctx context.Context)  /* (someExporter.Exporter, error) */ {
 }
 
 func newTraceProvider(exp sdktrace.SpanExporter) *sdktrace.TracerProvider {
-    // The service.name attribute is required.
-	resource := resource.NewWithAttributes(
-		semconv.SchemaURL,
-		semconv.ServiceNameKey.String("ExampleService"),
+	// Ensure default SDK resources and the required service name are set.
+	resource := resource.Merge(
```

Comment:
Please update to handle the error returned from `resource.Merge`.

Reply:
Is there a recommended way to do this? I feel like a conflict in `SchemaURL` when adding more resources should likely fail to initialize and panic the program. Otherwise, silently failing to add a resource on (or update a default one) would likely be tricker to hunt down.

Reply:
I went with a panic for now, but happy to adjust